manim/mobject/types/point_cloud_mobject.py

```diff
@@ -271,6 +271,26 @@ def generate_points(self):
 
 
 class Point(PMobject):
+    """A point.
+
+    Examples
+    --------
+
+    .. manim:: ExamplePoint
+        :save_last_frame:
+
+        class ExamplePoint(ThreeDScene):
```

Comment:
Just curious, is there a specific reason you chose `ThreeDScene` over `Scene`?

Reply:
It works with Scene too.
But while declaring a new Point, I have to pass a 3d point like location=[0,0,0]. If I pass [0,0], it does not work. So, I thought to go with ThreeDScene.

Reply:
I see. IMHO `ThreeDScene` is not really necessary here. Maybe just put coordinates like [x,y,0] (every coordinates that correspond to a scene are always 3d).

Reply:
Ok sure, will just use Scene.